app/models/medication_symptom.rb

```diff
@@ -0,0 +1,6 @@
+class MedicationSymptom < ApplicationRecord
+  validates :medication_id, :symptom_id, presence: true
```

Comment:
This PR should note that the purpose of this table is to create a record will all potential side effects a user might experience with a specific medication.

Also what happens if a user adds multiple meds and there are repeated side effects? I know in the drop down menu on the symptom log form the idea is to `unique` that list, but how are the repeated symptoms recorded in our actual tables?

Reply:
I'm not sure I understand what you're asking.  If a user records a symptom of `headache`, then that symptom will exist in the DB and (I don't imagine) that we were planning to create a new instance of a headache` symptom.  Then, if both adderall and zoloft have headache associated with them, then this table will create those relationships.